### ISSUE TITLE
HOTFIX: Swap out the Generate route to V2 to enable Data NFT minting watermarks + Use the new Data Marshal "router" URL

### ISSUE THREAD

**Issue by newbreedofgeek** [Itheum/data-dex#1047]
- We will use a new Data Marshal Generate route that does "watermark" protection on Data NFTs
- We will also use a new Data Marshal "router" URL in the mint form that allows for us to swap out to the new Data Marshal when it's available.

How to QA:
- Once the change is made, mint a few Data NFTs
- "View Data" on the Data DEX and also Explorer
- Test other features as normal (list, delist, update price etc)

Basically, once this change is made.. it should be the exact same user journey as before as it's just a security change but we have to make it it did not break anything.

**Comment by newbreedofgeek** [comment]
@fuzzyduke you can QA this here - https://github.com/Itheum/data-dex/pull/1048

**Comment by damienen** [comment]
Working so I am closing it. (to reopen if issues on stg)
